armi/plugins.py

```diff
@@ -600,10 +600,77 @@ def defineSystemBuilders() -> Dict[str, Callable[[str], "Composite"]]:
         """
 
 
-def getNewPluginManager() -> pluginManager.ArmiPluginManager:
+class UserPlugin(ArmiPlugin):
     """
-    Return a new plugin manager with all of the hookspecs pre-registered.
+    This is a variation on the usual ArmiPlugin that is meant to be read in during run time,
```

Comment:
change to be short summary 1-liner, blank line, more deets

Reply:
Done